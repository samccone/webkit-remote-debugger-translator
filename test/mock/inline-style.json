{
    "id": 40,
    "result": {
        "inlineStyle": {
            "cssProperties": [
                {
                    "implicit": false,
                    "name": "background",
                    "range": {
                        "endColumn": 0,
                        "endLine": 1,
                        "startColumn": 0,
                        "startLine": 0
                    },
                    "status": "active",
                    "text": "background:red",
                    "value": "red"
                },
                {
                    "name": "background-image",
                    "value": "initial"
                },
                {
                    "name": "background-repeat-x",
                    "value": "initial"
                },
                {
                    "name": "background-repeat-y",
                    "value": "initial"
                },
                {
                    "name": "background-attachment",
                    "value": "initial"
                },
                {
                    "name": "background-position-x",
                    "value": "initial"
                },
                {
                    "name": "background-position-y",
                    "value": "initial"
                },
                {
                    "name": "background-origin",
                    "value": "initial"
                },
                {
                    "name": "background-clip",
                    "value": "initial"
                },
                {
                    "name": "background-color",
                    "value": "red"
                },
                {
                    "name": "background-size",
                    "value": "initial"
                }
            ],
            "cssText": "background:red",
            "height": "",
            "range": {
                "endColumn": 0,
                "endLine": 1,
                "startColumn": 0,
                "startLine": 0
            },
            "shorthandEntries": [
                {
                    "name": "background",
                    "value": "red"
                }
            ],
            "styleId": {
                "ordinal": 0,
                "styleSheetId": "12"
            },
            "width": ""
        }
    }
}
