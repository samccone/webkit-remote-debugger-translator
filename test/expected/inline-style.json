{
    "id": 40,
    "result": {
        "inlineStyle": {
            "cssProperties": [
                {
                    "disabled": false,
                    "implicit": false,
                    "name": "background",
                    "range": {
                        "endColumn": 14,
                        "endLine": 0,
                        "startColumn": 0,
                        "startLine": 0
                    },
                    "text": "background:red",
                    "value": "red"
                },
                {
                    "name": "background-image",
                    "value": "initial"
                },
                {
                    "name": "background-repeat-x",
                    "value": "initial"
                },
                {
                    "name": "background-repeat-y",
                    "value": "initial"
                },
                {
                    "name": "background-attachment",
                    "value": "initial"
                },
                {
                    "name": "background-position-x",
                    "value": "initial"
                },
                {
                    "name": "background-position-y",
                    "value": "initial"
                },
                {
                    "name": "background-origin",
                    "value": "initial"
                },
                {
                    "name": "background-clip",
                    "value": "initial"
                },
                {
                    "name": "background-color",
                    "value": "red"
                },
                {
                    "name": "background-size",
                    "value": "initial"
                }
            ],
            "cssText": "background:red",
            "range": {
                "endColumn": 14,
                "endLine": 0,
                "startColumn": 0,
                "startLine": 0
            },
            "shorthandEntries": [
                {
                    "name": "background",
                    "value": "red"
                }
            ],
            "styleSheetId": "88928.9"
        }
    }
}
